### DISCUSSION TITLE
Unity 2018??

### DISCUSSION THREAD

**Discussion by jmjava** [ReactUnity/core#108] [Q&A] [answered]
I know it says 2019 is there any chance it will work in 2018?  I am involved with a game that is locked on 2018 at the moment

**Comment by gkurt** [comment, marked as answer]
I think Unity 2018 may not have the things required to run ReactUnity. Specifically the package manager, TMPro, and C# language features.

Tbh, I was planning on dropping support for Unity 2019 for a while. Maintaining this package for various Unity versions is the most difficult part, and I don't feel well about actively supporting Unity 2018.

**Reply by jmjava** [reply]
Understood.  No worries. I will wait for them to upgrade.  Thanks for the info
